iot/manager/manager.go

```diff
@@ -613,6 +614,38 @@ func setConfig(projectID string, region string, registry string, deviceID string
 	return response, err
 }
 
+// sendCommand sends a command to a device listening for commands
+func sendCommand(projectID string, region string, registry string, deviceID string, sendData string) (*cloudiot.SendCommandToDeviceResponse, error) {
```

Comment:
It's common to pass an `io.Writer` to write output to. Then, the line below would be `fmt.Fprintf(w, ...)`. That is much easier to test (can ensure the output is correct) and matches HTTP handlers.